ci/check_wheel_licenses.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+"""
+Check that all .whl files in the dist folder have the correct LICENSE files
+included.
+
+To run:
+    $ cp setup.cfg.template setup.cfg
+    $ python3 setup.py bdist_wheel
+    $ ./ci/check_wheel_licenses.py
+"""
+
+from pathlib import Path
+import sys
+import zipfile
+
+EXIT_SUCCESS = 0
+EXIT_FAILURE = 1
+
+project_dir = Path(__file__).parent.resolve().parent
+dist_dir = project_dir / 'dist'
+license_dir = project_dir / 'LICENSE'
+
+license_file_names = [Path(path).name
+                      for path in sorted(license_dir.glob('*'))]
```

Comment:
`Path.glob` returns `Path` objects already:
```suggestion
license_file_names = [path.name for path in sorted(license_dir.glob('*'))]
```